src/main/java/lotto/domain/Rank.java

```diff
@@ -2,32 +2,47 @@
 
 import java.util.Arrays;
 
+import lotto.domain.exception.IllegalMatchCountException;
+
 public enum Rank {
 	LOSS(0, 0),
 	FIFTH(3, 5_000),
-	FOURTH(4, 50_000),
+	FOURTH(4,50_000),
```

Comment:
사소하지만 컨벤션 관련해서 습관적으로 체크해주시는것도 도움이 될 것 같습니다 👍 
인텔리제이 + mac 을 사용하신다면 option + command + L 로 전체 소스코드를 정리할 수 있습니다 ~
```suggestion
	FOURTH(4, 50_000),
```

Reply:
항상 습관 처럼 option + command + L을 누르고 있는데 놓친 부분이 있었네요.
더 꼼꼼히 살피도록 하겠습니다. 👍